tests/zfs-tests/tests/functional/quota/quota.kshlib

```diff
@@ -57,7 +57,7 @@ function fill_quota
 	$FILE_WRITE -o create -f $MNTPT/$TESTFILE1 -b $BLOCK_SIZE \
 		 -c $write_size -d 0
 	zret=$?
-        [[ $zret -ne EDQUOT ]] && \
+        [[ $zret -ne $EDQUOT ]] && \
```

Comment:
I've just enabled `quota_002_pos`, `quota_004_pos` and `quota_005_pos` and they passed for me too.
That's on Gentoo, I tried with `DISKS` pointing to real 40GB disks and with `DISKS` unset.